src/components/PupilComponents/BrowseExamboardSelector/BrowseExamboardSelector.tsx

```diff
@@ -17,41 +16,53 @@ export const BrowseExamboardSelector = ({
   baseSlug,
   onClick,
   isLegacy,
+  phaseSlug,
 }: {
   examboards: ExamboardData[];
   baseSlug?: string;
   onClick?: (examboard: ExamboardData) => void;
   isLegacy: boolean;
-}) => (
-  <OakFlex $flexDirection={"column"} $gap={"space-between-s"}>
-    <OakHeading tag="h2">Choose an Examboard</OakHeading>
-    {(() => {
-      switch (true) {
-        case !!onClick:
-          return examboards.map((examboard) => (
-            <OakSecondaryButton
-              key={examboard.examboardSlug}
-              onClick={() => onClick && onClick(examboard)}
-            >
-              {examboard.examboard}
-            </OakSecondaryButton>
-          ));
-        case !!baseSlug:
-          return examboards.map((examboard) => (
-            <OakSecondaryButton
-              key={examboard.examboardSlug}
-              element="a"
-              href={resolveOakHref({
-                page: "pupil-unit-index",
-                programmeSlug: `${baseSlug}-${examboard.examboardSlug}${
-                  isLegacy ? "-l" : ""
-                }`,
-              })}
-            >
-              {examboard.examboard}
-            </OakSecondaryButton>
-          ));
-      }
-    })()}
-  </OakFlex>
-);
+  phaseSlug: PupilProgrammeListingData["programmeFields"]["phaseSlug"];
+}) => {
+  if (phaseSlug === "foundation" || !phaseSlug) {
+    throw new Error("Foundation phase is not supported");
+  }
+
+  return (
+    <>
+      {" "}
+      {(() => {
+        switch (true) {
+          case !!onClick:
+            return examboards.map((examboard) => (
+              <OakPupilJourneyYearButton
```

Comment:
all ive done here is swap out the oakSecondarybutton for the OakYearButton; and removed the container so the list items are rendered as the children to the programme options component